### ISSUE TITLE
[Bug]: newly created JSObject contains code from an existing JSObject 

### ISSUE THREAD

**Issue by bharath31** [appsmithorg/appsmith#18062]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When I create a new JSObject, I see code from an existing JSObect being copied over to the new one. 

### Steps To Reproduce

Steps to reproduce: 
1. Create a JSObject and add about 50-60 lines of code
2. Create a new JSObject 
3. Open JSObject1
4. Create a new JSObject 
5. Notice the code being copied 

### Public Sample App

https://app.appsmith.com/app/untitled-application-3/page1-6308d3d5573346502cb71806

### Issue video log

https://www.loom.com/share/71c41972b20641369b0e6abd004fba4a

### Version

Cloud

**Comment by bharath31** [comment]
Conversation: https://theappsmith.slack.com/archives/CGBPVEJ5C/p1667393828045139

@ApekshaBhosale should this be moved to the Integrations pod?

**Comment by ApekshaBhosale** [comment]
No. this is happening because creation is taking way too long right now. this will be resolved with 400 status issue which is handled by BE pod but we can have a backup fix from our pod only @bharath31
